source/time.tex

```diff
@@ -3662,13 +3634,13 @@
 
 \begin{itemdescr}
 \pnum
-\remarks
-This function does not participate in overload resolution unless
+\constraints
 \tcode{SourceClock::to_sys(t)} is well-formed.
-If \tcode{SourceClock::to_sys(t)}
-does not return \tcode{sys_time<Duration>},
-where \tcode{Duration} is a valid \tcode{chrono::duration} specialization,
-the program is ill-formed.
+
+\pnum
+\mandates
+\tcode{SourceClock::to_sys(t)} returns a \tcode{sys_time<Duration>},
+where \tcode{Duration} is a valid \tcode{chrono::duration} specialization.
```

Comment:
```suggestion
where \tcode{Duration} is a valid \tcode{duration} specialization.
```

Reply:
Hmm. [contents]p3 requires the `chrono::` here, but we're certainly not consistent about that.

Reply:
This is pre-existing, and the pre-existing wording is in line with [contents]p3. We should do something about this (we probably want an equivalent of [fs.req.namespace] for std::chrono), but not as part of applying this motion.

Reply:
Filed #3510.